src/Php73/Php73.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Symfony\Polyfill\Php73;
+
+/**
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Ion Bazan <ion.bazan@gmail.com>
+ *
+ * @internal
+ */
+final class Php73
+{
+    const NANO_IN_SEC = 1e9;
+    const NANO_IN_MSEC = 1000;
+    const MSEC_IN_SEC = 1e6;
+
+    private static $startAt = null;
+    private static $startAtArr = null;
+
+    /**
+     * @param bool $asNum
+     *
+     * @return array|float|int
+     */
+    public static function hrtime($asNum = false)
+    {
+        if (null === self::$startAt) {
+            self::$startAtArr = self::microtimeNumbers();
+            self::$startAt = self::$startAtArr[0] + (float) self::$startAtArr[1];
+            if (\PHP_INT_SIZE !== 4) {
```

Comment:
These checks look strange, what's the rationale?

Reply:
The idea was in the 64 bit version, I'm going to use it as a int, so I do the int conversion once here.